### PR TITLE
[DOCS-9044] enforcing monitor tags on synthetic tests

### PR DESCRIPTION
**Pull request by aliciascott**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/synthetics/platform/settings/_index.md [hunk 1]
```diff
@@ -47,7 +47,23 @@ On the [Synthetic Monitoring & Continuous Testing Settings page][1], you can acc
...
+Enforce tags for **usage attribution** on all tests:
+
+* On the Usage Attribution page, you can configure up to three tags by which to break down cost and usage attributes. Select **Enforce tags for usage attribution on all tests** to require that users enter all configured Usage Attribution tags when creating or editing Synthetic tests. With this setting enabled, users cannot save tests without entering all required tags. 
+
+Enforce required **monitor tag policies** on all tests:
+
+* On the [Synthetic Monitoring and Testings settings][20] page, select **Enforce required monitor tag policies on all tests** to require that users who set monitor tag policies will now have those tags enforced on Synthetic tests. With this setting enabled, users cannot save tests without entering all required tags.
+
+</br>
+
```
Comment (buraizu) [inline content/en/synthetics/platform/settings/_index.md:59]:
```suggestion
```

Comment (buraizu) [reply content/en/synthetics/platform/settings/_index.md:59]: This seems to be preventing the numbered list below from being indented (which it looks like it's intended to be).

If that's **not** the intended effect, I think you can update your `</br>` tag to just `<br>` or `<br/>`, as it's a self-closing tag.

**Review by buraizu — CHANGES_REQUESTED** [review]
Requesting some minor updates to address the Vale warnings, as well as some formatting suggestions. Let me know any feedback!

File: content/en/synthetics/platform/settings/_index.md (hunk 1 above)
Comment (aliciascott) [reply content/en/synthetics/platform/settings/_index.md]: thanks @buraizu ! Took a bit of fiddling but I think I got it now, can you re-review ?

File: content/en/synthetics/platform/settings/_index.md [hunk 2]
```diff
@@ -47,7 +47,27 @@ On the [Synthetic Monitoring & Continuous Testing Settings page][1], you can acc
...
 ### Enforced tags settings
 
-On the Usage Attribution page, you can configure up to three tags by which to break down cost and usage attributes. Select **Enforce tags for usage attribution on all tests** to require that users enter all configured Usage Attribution tags when creating or editing Synthetic tests. With this setting enabled, users cannot save tests without entering all required tags.
+#### Enforce tags for **usage attribution** on all tests
+
+* On the Usage Attribution page, you can configure up to three tags by which to break down cost and usage attributes. Select **Enforce tags for usage attribution on all tests** to require that users enter all configured Usage Attribution tags when creating or editing Synthetic tests. With this setting enabled, users cannot save tests without entering all required tags. 
+
+#### Enforce required **monitor tag policies** on all tests
+
+* On the [Synthetic Monitoring and Testing settings][20] page, select **Enforce required monitor tag policies on all tests** to require that user-defined monitor tag policies are enforced on Synthetic tests. With this setting enabled, users cannot save tests without entering all required tags.
```
Comment (buraizu) [inline content/en/synthetics/platform/settings/_index.md:56]:
```suggestion
On the [Synthetic Monitoring and Testing settings][20] page, select **Enforce required monitor tag policies on all tests** to require that user-defined monitor tag policies are enforced on Synthetic tests. With this setting enabled, users cannot save tests without entering all required tags. 
```

File: content/en/synthetics/platform/settings/_index.md [hunk 3]
```diff
@@ -47,7 +47,27 @@ On the [Synthetic Monitoring & Continuous Testing Settings page][1], you can acc
 
 ### Enforced tags settings
 
-On the Usage Attribution page, you can configure up to three tags by which to break down cost and usage attributes. Select **Enforce tags for usage attribution on all tests** to require that users enter all configured Usage Attribution tags when creating or editing Synthetic tests. With this setting enabled, users cannot save tests without entering all required tags.
+#### Enforce tags for **usage attribution** on all tests
+
+* On the Usage Attribution page, you can configure up to three tags by which to break down cost and usage attributes. Select **Enforce tags for usage attribution on all tests** to require that users enter all configured Usage Attribution tags when creating or editing Synthetic tests. With this setting enabled, users cannot save tests without entering all required tags.
```
Comment (buraizu) [inline content/en/synthetics/platform/settings/_index.md:52]:
```suggestion
On the Usage Attribution page, you can configure up to three tags by which to break down cost and usage attributes. Select **Enforce tags for usage attribution on all tests** to require that users enter all configured Usage Attribution tags when creating or editing Synthetic tests. With this setting enabled, users cannot save tests without entering all required tags. 
```

**Review by buraizu — APPROVED** [review]
Approving with a minor, optional formatting update. Since each section contains only one paragraph, I find the `-` character slightly visually distracting. Feel free to ignore this suggestion if that's intentional.

File: content/en/synthetics/platform/settings/_index.md (hunk 2 above)
Comment (aliciascott) [reply content/en/synthetics/platform/settings/_index.md:56]: that's a great point! I'll accept your suggestions, thanks!

**Pull request merged**